test/extensions/filters/http/common/fuzz/uber_per_filter.cc

```diff
@@ -0,0 +1,57 @@
+#include "envoy/extensions/filters/http/grpc_json_transcoder/v3/transcoder.pb.h"
+
+#include "extensions/filters/http/well_known_names.h"
+
+#include "test/extensions/filters/http/common/fuzz/uber_filter.h"
+#include "test/proto/bookstore.pb.h"
+
+// This file contains any filter-specific input clean-up needed in the generic filter fuzz target.
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace {
+
+void addFileDescriptorsRecursively(const Protobuf::FileDescriptor* descriptor,
```

Comment:
Nit: since this is Envoy code, prefer `&` over `*`.

Reply:
done